assist luv to find libuv headers/library
remove unecessary linking to libraries

diff -wpruN '--exclude=*.orig' a~/cmake/FindLibUV.cmake a/cmake/FindLibUV.cmake
--- a~/cmake/FindLibUV.cmake	1970-01-01 00:00:00
+++ a/cmake/FindLibUV.cmake	1970-01-01 00:00:00
@@ -29,42 +29,42 @@ set(LIBUV_INCLUDE_DIRS ${LIBUV_INCLUDE_D
 
 # Deal with the fact that libuv.pc is missing important dependency information.
 
-include(CheckLibraryExists)
-
-check_library_exists(dl dlopen "dlfcn.h" HAVE_LIBDL)
-if(HAVE_LIBDL)
-  list(APPEND LIBUV_LIBRARIES dl)
-endif()
-
-check_library_exists(kstat kstat_lookup "kstat.h" HAVE_LIBKSTAT)
-if(HAVE_LIBKSTAT)
-  list(APPEND LIBUV_LIBRARIES kstat)
-endif()
-
-check_library_exists(kvm kvm_open "kvm.h" HAVE_LIBKVM)
-if(HAVE_LIBKVM AND NOT CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
-  list(APPEND LIBUV_LIBRARIES kvm)
-endif()
-
-check_library_exists(nsl gethostbyname "nsl.h" HAVE_LIBNSL)
-if(HAVE_LIBNSL)
-  list(APPEND LIBUV_LIBRARIES nsl)
-endif()
-
-check_library_exists(perfstat perfstat_cpu "libperfstat.h" HAVE_LIBPERFSTAT)
-if(HAVE_LIBPERFSTAT)
-  list(APPEND LIBUV_LIBRARIES perfstat)
-endif()
-
-check_library_exists(rt clock_gettime "time.h" HAVE_LIBRT)
-if(HAVE_LIBRT)
-  list(APPEND LIBUV_LIBRARIES rt)
-endif()
-
-check_library_exists(sendfile sendfile "" HAVE_LIBSENDFILE)
-if(HAVE_LIBSENDFILE)
-  list(APPEND LIBUV_LIBRARIES sendfile)
-endif()
+#include(CheckLibraryExists)
+#
+#check_library_exists(dl dlopen "dlfcn.h" HAVE_LIBDL)
+#if(HAVE_LIBDL)
+#  list(APPEND LIBUV_LIBRARIES dl)
+#endif()
+#
+#check_library_exists(kstat kstat_lookup "kstat.h" HAVE_LIBKSTAT)
+#if(HAVE_LIBKSTAT)
+#  list(APPEND LIBUV_LIBRARIES kstat)
+#endif()
+#
+#check_library_exists(kvm kvm_open "kvm.h" HAVE_LIBKVM)
+#if(HAVE_LIBKVM AND NOT CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+#  list(APPEND LIBUV_LIBRARIES kvm)
+#endif()
+#
+#check_library_exists(nsl gethostbyname "nsl.h" HAVE_LIBNSL)
+#if(HAVE_LIBNSL)
+#  list(APPEND LIBUV_LIBRARIES nsl)
+#endif()
+#
+#check_library_exists(perfstat perfstat_cpu "libperfstat.h" HAVE_LIBPERFSTAT)
+#if(HAVE_LIBPERFSTAT)
+#  list(APPEND LIBUV_LIBRARIES perfstat)
+#endif()
+#
+#check_library_exists(rt clock_gettime "time.h" HAVE_LIBRT)
+#if(HAVE_LIBRT)
+#  list(APPEND LIBUV_LIBRARIES rt)
+#endif()
+#
+#check_library_exists(sendfile sendfile "" HAVE_LIBSENDFILE)
+#if(HAVE_LIBSENDFILE)
+#  list(APPEND LIBUV_LIBRARIES sendfile)
+#endif()
 
 if(WIN32)
   # check_library_exists() does not work for Win32 API calls in X86 due to name
diff -wpruN '--exclude=*.orig' a~/third-party/cmake/BuildLuarocks.cmake a/third-party/cmake/BuildLuarocks.cmake
--- a~/third-party/cmake/BuildLuarocks.cmake	1970-01-01 00:00:00
+++ a/third-party/cmake/BuildLuarocks.cmake	1970-01-01 00:00:00
@@ -201,6 +201,9 @@ if(USE_BUNDLED_BUSTED)
       if(MSVC)
         list(APPEND LUV_ARGS LIBUV_LIBDIR=${HOSTDEPS_INSTALL_DIR}/lib)
       endif()
+    else()
+      list(APPEND LUV_ARGS LIBUV_INCDIR="/opt/ooce/include")
+      list(APPEND LUV_ARGS LIBUV_LIBDIR="/opt/ooce/lib/amd64")
     endif()
     SET(LUV_PRIVATE_ARGS LUA_COMPAT53_INCDIR=${DEPS_BUILD_DIR}/src/lua-compat-5.3/c-api)
     add_custom_command(OUTPUT ${ROCKS_DIR}/luv
